install.sh

```diff
@@ -21,18 +21,47 @@ case $PLATFORM in
     ;;
 esac
 
+while getopts ":v:" opt; do
+  case $opt in
+    v)
+      VERSION=$OPTARG
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+    :)
+      echo "Option -$OPTARG requires an argument." >&2
+      exit 1
+      ;;
+  esac
+done
+
+if [ -n "$VERSION" ]; then
+  DESIRED_VERSION="tag/v${VERSION}"
```

Comment:
```suggestion
  REQUESTED_REF="tag/v${VERSION}"
```

this variable is holding git reference instead of version